scripts/src/main/resources/scripts/command/aws

```diff
@@ -42,7 +55,7 @@ function doPackageInstall()
 
 function doSetup() {
 
-  if [ ! -d "${AWS_HOME}" ] || [ "${1}" != "silent" ]
+  if [[ $(command -v aws) != "/c/Program Files/Amazon/AWSCLIV2/aws" ]] || [ "${1}" != "silent" ] 
```

Comment:
The first condition will never be met under MacOS because the given path is Windows specific. As a result AWS is being installed again and again every time devon aws is called. **command -v aws** returns different results (paths) depending on the OS and doSetup should be OS independent. So I think using the function doIsAwsInstalled would now make more sense here.

Reply:
If the doIsAwsInstalled function is used instead, it also works on Mac